src/auth/auth.service.ts

```diff
@@ -16,4 +21,22 @@ export class AuthService {
 
     return this.userModel.create(userObject);
   }
+
+  async login(userObjectLogin: LoginAuthDto) {
+    const { email, password } = userObjectLogin;
+    const findUser = await this.userModel.findOne({ email });
+    if (!findUser) throw new HttpException('User_Not_Found', 404);
```

Comment:
si bien esto funciona, las bunas practicas piden que usas el bloque if con llaves etc
```suggestion
    if (!findUser){ 
      throw new HttpException('User_Not_Found', 404);
    }
```
```suggestion
    if (!findUser) throw new HttpException('User_Not_Found', 404);
```